src/extension.ts

```diff
@@ -1,19 +1,16 @@
-"use strict";
-import { languages } from "vscode";
+import * as vscode from "vscode";
```

Comment:
Is this better than importing `{languages}`?

Reply:
`import * as vscode from "vscode";` is what is present by default when generating a new extension.

Have changed it back as it is potentially better when bundling to only import what is needed.